### DISCUSSION TITLE
No pass Percentage inside the chart - when 100% tests passed successfully

### DISCUSSION THREAD

**Discussion by varunmalik82** [test-results-reporter/reporter#122] [Q&A]
Pass percentage [100%] not displayed in the the chart when all tests passed successfully in the chart the way it is displayed when a certain percentage of tests passed. In the second example of the screen below, I was expecting to see 100% inside the circle. Without it, Chart seems incomplete to me 

<img width="775" alt="Screenshot 2023-02-15 at 6 11 00 PM" src="https://user-images.githubusercontent.com/17876410/219216422-f0402dc9-a501-46cf-9600-0da66c511f51.png">

**Comment by ASaiAnudeep** [comment]
Can you please share your test results file and report config file to further investigate the issue.

**Reply by varunmalik82** [reply]
@ASaiAnudeep - Please find both the files [here](https://github.com/varunmalik82/slack-report-issue)

<img width="1006" alt="image" src="https://user-images.githubusercontent.com/17876410/219390990-3c8c6f3a-19a7-49c7-9dca-f8cc9407b17b.png">

**Reply by varunmalik82** [reply]
Hi @ASaiAnudeep - did you get a chance to look into this issue?
